lib/truffle/digest.rb

```diff
@@ -136,17 +136,23 @@ def inspect
   class Class
     include Instance
 
-    def self.digest(message)
-      digest = new
+    def self.digest(message, *parameters)
+      digest = new(*parameters)
       digest.update message
       digest.digest
     end
 
-    def self.hexdigest(message)
-      digest = new
+    def self.hexdigest(*parameters, message)
+      digest = new(*parameters)
       digest.update message
       digest.hexdigest
     end
+
+    def self.base64digest(*parameters, message)
+      digest = new(*parameters)
+      digest.update message
+      digest.base64digest
+    end
```

Comment:
Just a note: this was already defined at line 29 above:
https://github.com/oracle/truffleruby/pull/1889/files#diff-048c3691437021dc9c9997d1a3aa7792R29

This file is pretty hard to follow due to reopening classes and part of the code from MRI.
Maybe I should try to convince MRI devs to define more of Digest in Ruby :)